### PR TITLE
convert telegram blog post to doc

### PR DESCRIPTION
**Pull request by tonypowa**
**What is this feature?**

- Adaptation of the [telegram integration blog post](https://grafana.com/blog/2023/12/28/how-to-integrate-grafana-alerting-and-telegram/) to a formal doc

- Plus a small typo fix in the [slack integration point doc](https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/integrations/configure-slack/)

**Why do we need this feature?**

Telegram contact point was missing in the docs
https://grafana.com./docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/

Please check that:
- [X] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Comment by tonypowa** [comment]
note: the fix for slack needs backport to current -v of docs

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ To create your Slack integration in Grafana Alerting, complete the following ste
 To add the contact point and integration you created to your default notification policy, complete the following steps.
 
 1. Navigate to **Alerts & IRM** -> **Alerting** -> **Notification policies**.
-1. In the **Default policy**, click the ellipsis icon (…) and then **Edit**,
+1. In the **Default policy**, click the ellipsis icon (…) and then **Edit**
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md:82]:
```suggestion
1. In the **Default policy**, click the ellipsis icon (…) and then **Edit**.
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## Before you begin
+
+### Telegram bot API token and chat ID
+
+To integrate Grafana with Telegram, you need to obtain a Telegram **bot API token** and a **chat ID** (i.e., the ID of the Telegram chat where you want to receive the alert notifications).
+
+### Set up your Telegram bot
+
+In order to send any messages over HTTP to Telegram, you need to create a [Telegram bot](https://core.telegram.org/bots/api). You will be able to associate this bot to your chats, and perform different actions with it, such as receiving alerts from Grafana.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:31]:
```suggestion
Create a [Telegram bot](https://core.telegram.org/bots/api). You can associate this bot to your chats and perform different actions with it, such as receiving alerts from Grafana.
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+To set up the bot, complete the following steps.
+
+1. **Open the Telegram app** on your device
+2. Find the Telegram bot named **BotFather**
+3. Type or press `/newbot`.
+4. Choose a name for the bot. It must end in **bot** or **\_bot**. E.g. "my_bot".
+5. **Copy the API token**
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:39]: add full stops in every step and number steps always with 1 1 1 1

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+### Chat ID
+
+Add the bot to a group chat by following the steps below. Once the bot is added to the chat, you will be able to route your alert notifications to that group.
+
+1. In the Telegram app, **open a group or start a new one**
+2. Search and **add the bot to the group**
+3. **Interact with the bot** by sending a dummy message that starts with "`/`". E.g. `/hola @bot_name`
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:47]: same re full stops and numbering

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+           "title": "Tony and Hello world bot",
+           "type": "group",
+   ...
+   ```
+
+7. Copy the value of the `“id”` that appears under `“chat”`.
+
+## Procedure
+
+To create your Telegram integration in Grafana Alerting, complete the following steps.
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:73]: for this section follow the same pattern as slack: https://grafana.com/docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/integrations/configure-slack/

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+1. Click **Save contact point**.
+
+{{< figure src="/media/blog/telegram-grafana-alerting/telegram-grafana-alerting-contact-point-.png" alt="A screenshot showing how to save a new contact point in Grafana Alerting. " >}}
+
+## Next steps
+
+To add the contact point and integration you created to your default [notification policy][notification-policy], complete the following steps.
+
+1. Navigate to **Alerts & IRM** -> **Alerting** -> **Notification policies**.
+1. In the **Default policy**, click the ellipsis icon (…) and then **Edit**
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:95]:
```suggestion
1. In the **Default policy**, click the ellipsis icon (…) and then **Edit**.
```

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+1. Change the default policy to the contact point you created.
+1. Click **Update default policy**.
+
+**Note:**
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:99]: remember the admonition format for notes:)

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+If you have more than one contact point, add a new notification policy rather than edit the default one, so you can route specific alerts to Slack. For more information, refer to [Notification policies][nested-policy].
+
+{{% docs/reference %}}
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:102]: the linking has changed in alerting -- check another topic to follow the format (we now add the links to the front matter)

File: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+## Configure Telegram for Alerting
+
+Use the Grafana Alerting - Telegram integration to send [Telegram](https://telegram.org/) notifications when your alerts are firing.
+
+## Before you begin
+
+### Telegram bot API token and chat ID
+
+To integrate Grafana with Telegram, you need to obtain a Telegram **bot API token** and a **chat ID** (i.e., the ID of the Telegram chat where you want to receive the alert notifications).
```
Comment (brendamuir) [inline docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:27]: where?

Comment (tonypowa) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:27]: i can add in-doc links so users don't get lost.

Comment (brendamuir) [reply docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md:27]: you can ignore this. i see you describe it below:)

**Review by brendamuir — APPROVED** [review]

**Pull request merged**